process/transaction/interceptor_test.go

```diff
@@ -51,7 +54,11 @@ func TestNewTxInterceptor_NilTransactionPoolShouldErr(t *testing.T) {
 	addrConv := &mock.AddressConverterMock{}
 	keyGen := &mock.SingleSignKeyGenMock{}
 	oneSharder := mock.NewOneShardCoordinatorMock()
-	txValidator := &mock.TxValidatorStub{}
+	txValidator := &mock.TxValidatorStub{
```

Comment:
Should have a function that returns a mock tx validator with its func members set

Reply:
Done.